modules/images-triggering-updates-imagestream-changes-kubernetes.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/managing_images/triggering-updates-on-imagestream-changes.adoc
+
+
+[id="images-triggering-updates-imagestream-changes-kubernetes_{context}"]
+= Kubernetes resources
+
+Unlike DeploymentConfigs and BuildConfigs, which include as part of their API definition a set of fields for controlling triggers, Kubernetes resources do not have fields for triggering. Instead, {product-title} uses annotations to allow users to request triggering.
```

Comment:
Need to follow up on style changes for DeploymentConfigs, BuildConfigs, CronJobs, StatefulSets, DaemonSets, ReplicaSets, and ReplicationControllers.